### ISSUE TITLE
Automaticly adding players, roles to DB

### ISSUE THREAD

**Issue by 1teacherr1** [AdKats/AdKats#748]
Hello,

Going by the advise that Prohpet  from Adgamers.com forum gave me I'm asking about possibility of adding that thing, I'll just copy my post here.

"Its AdKats role, i've mention BFPanel beacuse its possible to manage those players/soldiers/role by panel.
And yea you are probably right about those numbers and lags.
How about auto. assign players to a (already existing AdKats Role) after reaching X rep? From what i know every player is assignet to "Guest role" anyway so there is no need to add those players. Editing the "Default Guest" with InsaneLimits should work without overloading DB too much. 
 
But for those regular players who are playing on the server everyday and they are making the progression/regression in the reputation I'd like to create auto assigment. That should be best example what i mean:
New player join server for 1st time - His reputation is 0 (Default Guest) -> He reach the 250 reputation and now he is automaticly assigned to (Regular Player role) - by he I mean the Player AND ingame Soldier name. After he reach 500 reputation his role is changed to (Veteran Player) (the soldier assignation to X USER is needed only once) same way. In the case if he would fall down from 250 reputation to 0 his user/soldier would be completely removed from DB (to prevent issues with lags and wasting space in DB) and creating it again if he would reach 250rep one more time (and repeat)"

**Comment by iraqiboy90** [comment]
Nice idea, but it's always nice to have such features as off by default :)

**Comment by 1teacherr1** [comment]
Yea ofcourse, this can cause db overloading with too much data so I never thought about forcing to use option like those. This would be really helpful with the project that I'm trying to create. This specific thing would allow me to save a lot of time, now to add every user/role/assign soldier to id I have to use either procon tabels or BFadminCP panel, but still its time consuming. I'd like to have thing like this beacuse I'm trying to create many dependencies for reputation. For example people with XXX positive reputation will be allowed to use specific items, which are normally disabled for everyone.

**Comment by ColColonCleaner** [comment]
This could be another system added to the engine. All AdKats 7 issues need to be recreated on the engine repository in the proper order. If you are still interested in this, add it there.
